docs/src/components/MainSection.js

```diff
@@ -5,14 +5,15 @@ import Card from './Card.js';
 import MainSectionSubsection from './MainSectionSubsection.js';
 
 type Props = {|
-  children: Node,
+  children?: Node,
```

Comment:
this component won't always have children?

Reply:
No, sometimes subsections are only titles and descriptions